### ISSUE TITLE
Admin Panel Into a server.

### ISSUE THREAD

**Issue by hunterpl** [huytd/agar.io-clone#146]
Hello Dear Developer

Do you have plan to make a panel admin with simple commands into chat?
example:

1. kick with own reason + time
2. ban with own reason + time
3. IP Ban
4. Maintenance function poke window on the screen with comming maintenance and add own reason. 

Best Regards
Mateusz

**Comment by huytd** [comment]
Good idea! :+1:

**Comment by hunterpl** [comment]
Kick function working but I can kick myself it shouldn't kick the Administrator.

**Comment by themuddfamily2** [comment]
working on a fix for kicking admins.

**Comment by hunterpl** [comment]
How can I kick the unnamed cell?

**Comment by themuddfamily2** [comment]
erh pretty sure -kick will work :P

**Comment by themuddfamily2** [comment]
literally just put -kick

**Comment by hunterpl** [comment]
@themuddfamily2 nope, you just don't understand me, i'm know how to kick the users but if someone join to the server without nickname so just he is unnamed cell how can I kick him?

**Comment by IgorAntun** [comment]
As @themuddfamily2 said, just use `-kick`, with no following arguments, it should kick any player with a blank name.

**Comment by hunterpl** [comment]
@IgorAntun it doesn't work how he wrote that. I just make "-kick " so -kick with spacebar and then working.

**Comment by bigfoot90** [comment]
+1

**Comment by Crasha** [comment]
Maybe instead of leaving "-kick" empty, unnamed players should increment if there is more than one. E.g...

Unnamed1
Unnamed2
Etc...

Therefore "-kick" can always have a parameter instead of being left blank.

**Comment by jamessilva** [comment]
how do i  become an admin, please help fast

**Comment by jamessilva** [comment]
How do i become an admin, please help quickly, thx

**Comment by jamesmou** [comment]
How do i become an admin, made my own on the heroku, please help, thx

**Comment by abalabahaha** [comment]
@jamesmou You're not commenting in the right places. Use `-login PASSWORD` with the password you set in the config.

**Comment by Grapenut21** [comment]
where is the config??

**Comment by Grapenut21** [comment]
@huytd @abalabahaha  where is the config??

**Comment by xwardbm** [comment]
how do i  become an admin
